tools/dotcms-cli/api-data-model/src/main/java/com/dotcms/api/LanguageAPI.java

```diff
@@ -0,0 +1,43 @@
+package com.dotcms.api;
+
+import com.dotcms.api.provider.DefaultResponseExceptionMapper;
+import com.dotcms.api.provider.DotCMSClientHeaders;
+import com.dotcms.model.ResponseEntityView;
+import com.dotcms.model.language.Language;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.eclipse.microprofile.openapi.annotations.tags.Tags;
+import org.eclipse.microprofile.rest.client.annotation.RegisterClientHeaders;
+import org.eclipse.microprofile.rest.client.annotation.RegisterProvider;
+
+@Path("/v2/languages")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Tags(
+        value = { @Tag(name = "Language")}
+)
+@RegisterClientHeaders(DotCMSClientHeaders.class)
+@RegisterProvider(DefaultResponseExceptionMapper.class)
+public interface LanguageAPI {
```

Comment:
Let's add some Javadoc to all classes please.

Reply:
Done!